src/openrct2/world/scenery.h

```diff
@@ -80,6 +80,10 @@ typedef enum {
     SMALL_SCENERY_FLAG27 = (1 << 27),                           // 0x8000000
 } SMALL_SCENERY_FLAGS;
 
+typedef enum {
```

Comment:
This is a map element small scenery flag the union is defined in the map_element file. Should probably move this over to there. Also its not clear that this is for small scenery map elements